Bio/Application/__init__.py

```diff
@@ -729,6 +732,12 @@ def _escape_filename(filename):
 
     Note this will not add quotes if they are already included:
 
+    Note the function is more generic than the name suggests, since
+    it is used to add quotes around string arguments containing spaces
+    that are not filenames, e.g. see issue #2055 on github. In the future
+    this function might be renamed like 'def _quote_spaces(spaced_string):'
+    or similar, assuming it does not break compatability.
```

Comment:
Since ``_escape_filename`` has a leading underscore, it is private. You could rename it as part of this PR.

Reply:
Or, shorten the docstring to finish:

```
    Note the function is more generic than the name suggests, since
    it is used to add quotes around any string arguments containing spaces.
```

Also can you move this below the doctest example, as right now the trailing colon on the previous paragraph looks wrong.